### DISCUSSION TITLE
EMP 3.0 设想、调研、讨论

### DISCUSSION THREAD

**Discussion by ckken** [empjs/emp#341] [Ideas]
目前emp 3.0(暂定命名) 处在前期预研阶段 可能会进行相关调整 
## 目前设想清单
> 持续收集各方意见
+ 聚焦 React + TS 的技术推进 
+ 基于 Module Federation 建立完善的微前端生态 如 (跨模块热更等问题)
+ 构建速度显著提升 (目前提升 15~20倍) 
+ 建立 基站 与 主站等基础模版 得以更加便捷的开发 和引用 (物料库模型)

**Comment by codcer** [comment]
会集成vite吗?
